File: programmers/lev1/12901.py
def solution(a, b):
    answer = ''

    week=["FRI","SAT","SUN","MON","TUE","WED","THU"]
    mon=[31,29,31,30,31,30,31,31,30,31,30,31]

    answer=week[(sum(mon[:a-1])+b-1)%7]
    return answer

a=1
b=1
solution(a,b)

File: java/src/programmers/year2023/lv1/_12901.java
package programmers.year2023.lv1;

public class _12901 {
    public static void main(String[] args) {

    }

    public static String solution(int a, int b) {
        String[] day = {"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"};
        int[] month = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};

        int date = 0;
        for (int i = 0; i < a - 1; i++) {
            date += month[i];
        }

        date += b - 1;

        return day[date % 7];
    }
}
